### PR TITLE
Consolidate stiserr.h, wf3err.h, acserr.h, & the errors in hstcal/inc…

### LINKED ISSUE #95: Consolidate stiserr.h, wf3err.h, & acserr.h to hstcal/include/err.h

### PR DESCRIPTION
**Pull request by jamienoss**
…lude/imphttab.h to hstcal/include/hstcalerr.h

Fixes #95 

Signed-off-by: James Noss <jnoss@stsci.edu>

### CODE REVIEW

**Comment by sosey** [comment]
I would say it's too late to start inserting package changes to this build. We should keep this on hold until after the delivery and then do package restructuring if it's been requested and the resources approved by @stsci-hack

**Comment by jamienoss** [comment]
See old PR #94 for some comments.

File: pkg/stis/calstis/lib/mkpkg [hunk 1]
```diff
@@ -11,67 +11,67 @@ install:
 	;
 
 calstis.a:
-	add2d.c		<c_iraf.h> <hstio.h> ../stis.h ../stiserr.h ../stisdef.h
```
Comment (jamienoss) [inline pkg/stis/calstis/lib/mkpkg]: @jhunkeler I think the other relative paths are incorrect(?) Is this file even still needed or can we just delete it?

**Comment by jamienoss** [comment]
@pllim and/or @jhunkeler can you please review this.

**Comment by jhunkeler** [comment]
So nothing in the code directly references `pkg/stis/calstis/lib/mkpkg`. The only script of this nature generated at compile-time is `cfitsio/mkpkg`, *and* only because it was a direct port from IRAF, however it isn't used by anything.

Restructuring efforts aside -- `pkg/stis/calstis/lib/mkpkg` and `cfitsio/mkpkg` are dead files and should be purged.

**Comment by jamienoss** [comment]
@pllim can you please spend a quick 5min and look over this for me, cheers.

**Comment by pllim** [comment]
I looked at ACS changes and they seem okay. Extrapolating that to WFC3 and STIS, should be okay. I do like how all the error codes are now properly sorted in the same file.

One way to make sure this works is to install this branch, and call one of the CALs with a bad input (e.g., trying to run CALACS with undefined "jref" or trying to run CALWF3 on ACS input file), then make sure the proper error is returned.

**Comment by jamienoss** [comment]
@pllim thank you. This sort of change requires only a build and not a run. As long as the numbers match up, which can be shown by reviewing the git comparison, the only thing left is that something hasn't been left out of the new consolidated header, if this were the case the build would fail.

**Pull request merged**